src/en/aurora/src/eu/kanade/tachiyomi/extension/en/aurora/Aurora.kt

```diff
@@ -0,0 +1,227 @@
+                }
+
+            foundChapters.addAll(pagesAsChapters)
+
+            // get a potential next page of the chapter overview
+            val nextPageNavUrl: String =
+                currentPage.select("li.paginav-next")
+                    .select("a[href]").attr("href")
+
+            // check if a next page actually exits and if not exit
+            return if (nextPageNavUrl.isEmpty()) {
+                foundChapters
+            } else {
+                fetchChapterListTR(nextPageNavUrl, foundChapters)
+            }
+        }
+
+        return Observable.just(fetchChapterListTR(manga.url, mutableListOf()))
+    }
+
+    override fun fetchImageUrl(page: Page): Observable<String> {
+        return Observable.just(page.imageUrl)
+    }
+
+    override fun imageUrlParse(response: Response): String = throw Exception("Not used")
+
+    override fun latestUpdatesParse(response: Response): MangasPage = throw Exception("Not used")
+
+    override fun latestUpdatesRequest(page: Int): Request = throw Exception("Not used")
+
+    override fun fetchMangaDetails(manga: SManga): Observable<SManga> {
+        val chapterNrRegex = "[0-9]+".toRegex()
+        val chapterNr = chapterNrRegex.find(manga.title)?.value?.toFloat() ?: 0f
+
+        val updatedManga = SManga.create().apply {
+            url = manga.url
+            title = manga.title
+            artist = authorName
+            author = authorName
+            description = getAuroraDescription()
+            genre = auroraGenre
+            status = getChapterStatusForChapter(chapterNr)
+            thumbnail_url = manga.thumbnail_url
+        }
+        return Observable.just(updatedManga)
+    }
+
+    /**
+     * @param chapter chapter the status should be fetched for
+     * @return the status of the chapter (as Enum value of SManga because chapters are mangas)
+     */
+    private fun getChapterStatusForChapter(chapter: Float): Int {
+        val newestPage = client.newCall(GET(baseUrl)).execute().asJsoup()
+        val postTitle = newestPage.select(".post-title").text()
+        // title is "<arc>.<chapter>.<page>"
+        val chapterOfNewestPage = postTitle.split(".")[1].toFloat()
+        return if (chapter >= chapterOfNewestPage) SManga.UNKNOWN else SManga.COMPLETED
+    }
+
+    /**
+     * This method requests the description of the aurora comic from the about page of the website.
+     * @return the description of Aurora in the about page
+     */
+    private fun getAuroraDescription(): String {
+        // only the first three of these paragraphs actually matter at the time of writing (2021)
+        val descriptionDoc = client.newCall(GET("$baseUrl/about")).execute().asJsoup()
+        val descriptionParagraphs = descriptionDoc
+            .select("div.entry")
+            .select("p").take(3)
+        return descriptionParagraphs
+            .map { paragraph -> paragraph.text() }
+            .reduce { text, paragraph -> "$text $paragraph" }
+    }
```

Comment:
You could just hardcode this.

```suggestion
    private val auroraDescription = """
    Aurora is a fantasy webcomic (updates M/W/F) written and illustrated by Red, better known for her work on the YouTube channel “Overly Sarcastic Productions.” It’s been in the works for over a decade, and she’s finally decided to stop putting it off.

    If you’d like to discuss the comic, it now has a subreddit, as well as a dedicated twitter and a tumblr where you can ask questions. There’s also a dedicated room on the channel discord for conversations about it!

    Find Red’s general ramblings on Twitter, alongside her cohost Blue, at OSPYouTube.
    """.trimIndent()
```